README.md

```diff
@@ -1,2 +1,21 @@
-# hw01
-Repository for homework 01
+![Image of Me](GitHubPhoto.jpg)
+
+# Eitan's Homework #1: A Brief Biography
+
+My story begins in Brooklyn on April 2nd, 1998. I went to to Poly Prep Country Day School in Brooklyn where I was primarily engaged in high school debate. Now a Junior at the University of Chicago majoring in *Political Science* and *Psychology*. 
```

Comment:
I need to stop looking at the birth years of my students. It reminds me I am old :(